WooCommerce/Classes/POS/Utils/POSStockFormatter.swift

```diff
@@ -18,15 +28,38 @@ struct POSStockFormatter {
         }
     }
 
+    private static func label(for key: String) -> String {
+        switch key {
+        case "instock":
```

Comment:
`StockStatusKey` happens to be in Networking at the moment, so we cannot use it directly in this target. I'll update the usage from String to an enum with WOOMOB-517 when testing/handling undocumented stock statuses.